config/initializers/geocoder.rb

```diff
@@ -1,10 +1,14 @@
+db_file = Rails.env.production? ?
+  File.join('/usr', 'share', 'GeoIP', 'GeoLite2-City.mmdb') :
+  Rails.root.join('vendor', 'GeoLite2-City.mmdb')
```

Comment:
@ErikSchierboom Two changes: 
1. city not country DB
2. Local location has changed to vendor (so it can be consistent across OSes)

The dockerfile will need updating with the new db.

Reply:
(I've updated the Dockerfile too)